src/virtual_elements.js

```diff
@@ -145,16 +146,15 @@ var elementOpen = function(tag, key, statics, var_args) {
   if (attrsChanged) {
     var newAttrs = data.newAttrs;
 
-    for (var attr in newAttrs) {
```

Comment:
This is one of the optimizations I found in https://github.com/google/incremental-dom/pull/95.